fixtures/src/main.js

```diff
@@ -0,0 +1,110 @@
+import ControlledNumber from './components/controlled/number';
+import UncontrolledNumber from './components/uncontrolled/number';
+import ControlledGroupedCharacter from './components/controlled/grouped-character';
+import UncontrolledGroupedCharacter from './components/uncontrolled/grouped-character';
+import ControlledSimpleColorPicker from './components/controlled/simple-color-picker';
+import UncontrolledSimpleColorPicker from './components/uncontrolled/simple-color-picker';
+import ControlledDate from './components/controlled/date';
+import UncontrolledDate from './components/uncontrolled/date';
+import ControlledSelect from './components/controlled/select';
+import UnControlledSelect from './components/uncontrolled/select';
+import ControlledDateRange from './components/controlled/date-range';
+import UncontrolledDateRange from './components/uncontrolled/date-range';
+import UncontrolledCheckboxGroup from './components/uncontrolled/checkbox-group';
+import ControlledCheckboxGroup from './components/controlled/checkbox-group';
+import UncontrolledCheckbox from './components/uncontrolled/checkbox';
+import ControlledCheckbox from './components/controlled/checkbox';
+
+const routes = {
+  'radio-button': {
+    description: 'Radio Button',
+    controlled: ControlledRadioButton,
+    uncontrolled: UncontrolledRadioButton
+  },
+  switch: { description: 'Switch', controlled: ControlledSwitch, uncontrolled: UncontrolledSwitch },
+  date: { description: 'Date', controlled: ControlledDate, uncontrolled: UncontrolledDate },
+  dateRange: { description: 'Date Range', controlled: ControlledDateRange, uncontrolled: UncontrolledDateRange },
+  decimal: { description: 'Decimal', controlled: ControlledDecimal, uncontrolled: UncontrolledDecimal },
+  textarea: { description: 'Text Area', controlled: ControlledTextArea, uncontrolled: UncontrolledTextArea },
+  textbox: { description: 'Text Box', controlled: ControlledTextbox, uncontrolled: UncontrolledTextbox },
+  number: { description: 'Number', controlled: ControlledNumber, uncontrolled: UncontrolledNumber },
+  'grouped-character': {
+    description: 'Grouped Character',
+    controlled: ControlledGroupedCharacter,
+    uncontrolled: UncontrolledGroupedCharacter
+  },
+  'simple-color-picker': {
+    description: 'Simple Color Picker',
+    controlled: ControlledSimpleColorPicker,
+    uncontrolled: UncontrolledSimpleColorPicker
+  },
+  select: {
+    description: 'Select',
+    controlled: ControlledSelect,
+    uncontrolled: UnControlledSelect
+  },
+  checkboxgroup: {
+    description: 'Checkbox Group',
+    uncontrolled: UncontrolledCheckboxGroup,
+    controlled: ControlledCheckboxGroup
+  },
+  checkbox: {
+    description: 'Checkbox',
+    uncontrolled: UncontrolledCheckbox,
+    controlled: ControlledCheckbox
+  }
+};
+
+
+startRouter(
+  [
+    <Route
+      path='/' component={ Index }
+      routes={ routes }
+    />,
+    <Route path='/' component={ Chrome }>
```

Comment:
I think we need to add `key` props to these `<Route>` components. We're currently getting this warning: :warning:
```
Warning: Each child in a list should have a unique "key" prop.
Check the top-level render call using <Router>.
    in Route
```

Reply:
Yeah I've added keys to the routes that were missing them

Reply:
Thanks for fixing this in 6b8ce63 :white_check_mark: